broker/src/main/java/org/apache/rocketmq/broker/processor/PullMessageProcessor.java

```diff
@@ -87,8 +87,8 @@ private RemotingCommand processRequest(final Channel channel, RemotingCommand re
 
         response.setOpaque(request.getOpaque());
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("receive PullMessage request command, {}", request);
+        if (log.isDebugEnabled()) {
```

Comment:
How is it related to this PR? Please see the title.
That is from the original code, probably you can find ten more small improvements in files I edited but lines I haven't even touched ;)